src/qt/optionsdialog.cpp

```diff
@@ -50,6 +50,9 @@ OptionsDialog::OptionsDialog(QWidget *parent, bool enableWallet) :
     /* Network elements init */
 #ifndef USE_UPNP
     ui->mapPortUpnp->setEnabled(false);
+#endif
+#ifndef USE_NATPMP
+    ui->mapPortNatpmp->setEnabled(false);
```

Comment:
Nit: not s/false/USE_NATPMP ?

Reply:
It won't compile if `USE_NATPMP` is undefined. Did I understand you correctly?

Reply:
Oh my mistake; I read that as ifdef.